.github/workflows/_rust_tests.yml

```diff
@@ -204,6 +204,6 @@ jobs:
           cargo nextest run --no-fail-fast --test-threads 8 --package iota-graphql-e2e-tests --features pg_integration
           cargo nextest run --no-fail-fast --test-threads 1 --package iota-cluster-test --test local_cluster_test --features pg_integration
           cargo nextest run --no-fail-fast --test-threads 1 --package iota-indexer --test ingestion_tests --features pg_integration
-          # Iota-indexer's RPC tests, which depend on a shared runtime, are incompatible with nextest due to its process-per-test execution model.
+          # IOTA-indexer's RPC tests, which depend on a shared runtime, are incompatible with nextest due to its process-per-test execution model.
```

Comment:
I don't know about this one. Seems like this should be all lower if anything

Reply:
fixed